drivers/serial/uart_rpi_pico.c

```diff
@@ -40,6 +52,9 @@ static void uart_rpi_poll_out(const struct device *dev, unsigned char c)
 static int uart_rpi_init(const struct device *dev)
 {
 	const struct uart_rpi_config *config = dev->config;
+	uart_inst_t * const uart_inst = config->uart_dev;
```

Comment:
To keep the tree bisect-able, this should be in the previous commit where the call to `uart_init()` was added.

Reply:
Yes, you are right, it was my mistake. It is okay now.